api/circulation_exceptions.py

```diff
@@ -79,7 +79,28 @@ class OutstandingFines(CannotLoan):
     """The patron has outstanding fines above the limit in the library's 
     policy."""
     status_code = 403
+    
+class AuthorizationExpired(CannotLoan):
+    """The patron's authorization has expired."""
+    status_code = 403
 
+    def as_problem_detail_document(self, debug=False):
+        """Return a suitable problem detail document."""
+        return EXPIRED_CREDENTIALS
+    
+class AuthorizationBlocked(CannotLoan):
```

Comment:
Currently we don't know how to read from the ILS whether the patron's card is blocked, but it's coming soon, so I figured I'd add the exception and the problem detail document as long as I'm dealing with reasons why the patron might not have borrowing privileges.